examples/scaling-criteo/04-Triton-Inference-with-Merlin-Models-TensorFlow.ipynb

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
<p> let's use past tense for all the verbs: "First, NVTabular created features, and afterwards Merlin Models TensorFlow generated the predictions for the processed data."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>